ftw/workspace/tests/test_workspace.py

```diff
@@ -0,0 +1,41 @@
+from Products.CMFCore.utils import getToolByName
+from ftw.builder import Builder
+from ftw.builder import create
+from ftw.workspace.testing import FTW_WORKSPACE_INTEGRATION_TESTING
+from plone.app.testing import login
+from plone.app.testing import setRoles
+from plone.app.testing import TEST_USER_ID
+from unittest2 import TestCase
+
+
+class TestWorkspace(TestCase):
+
+    layer = FTW_WORKSPACE_INTEGRATION_TESTING
+
+    def setUp(self):
+        self.portal = self.layer['portal']
+        setRoles(self.portal, TEST_USER_ID, ['Manager'])
+
+    def test_searchable_text_contains_workspace_creator(self):
+        """
+        The full name of the creator of a workspace has to be in the
+        searchable text index so the workspace overview listing can
+        be filtered by creator.
+        """
+        folder = create(Builder('folder'))
```

Comment:
I think when a title is set here, it will appear in the (default) searchable text.
In the assertion below it would verify that the words are correctly separated.